app/views/layouts/_nav.html.erb

```diff
@@ -23,7 +23,7 @@
           <% if @selected_branding_group.present? && @selected_branding_group.external_link_url %>
             <li><%= link_to "<span class='fas fa-fw fa-globe'></span> ".html_safe + @selected_branding_group.external_link_description.html_safe, @selected_branding_group.external_link_url, target: '_blank', class: 'check-upload' %></li>
           <% end %>
-          <li><%= link_to "<span class='fas fa-fw fa-info-circle'></span> Documentation/Wiki".html_safe, 'https://github.com/broadinstitute/single_cell_portal/wiki', target: '_blank' %></li>
+          <li><%= link_to "<span class='fas fa-fw fa-info-circle'></span> Documentation/Zendesk Guide".html_safe, 'https://singlecell.zendesk.com/hc/en-us', target: '_blank' %></li> 
```

Comment:
Not necessarily advocating for this, but do we really want to call it the "Zendesk Guide" in the UI?  Wouldn't "User Guide" be more generic, and a term people are familiar with?